.github/workflows/ci.yml

```diff
@@ -24,12 +24,19 @@ jobs:
           cmake -S $GITHUB_WORKSPACE -B $GITHUB_WORKSPACE/build-linux
           -DCMAKE_BUILD_TYPE=Release
           -DBUILD_EXAMPLES=ON
-          -DCMAKE_CXX_CLANG_TIDY=clang-tidy
           -DCMAKE_PREFIX_PATH=$BOOST_ROOT_1_72_0
+          -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
 
       - name: Build
         run: cmake --build $GITHUB_WORKSPACE/build-linux --parallel 3
 
+      - name: Clang Tidy
+        run: |
+          export MODIFIED_FILES=$(git diff -U0 HEAD^ --name-only |grep -E "\.(cpp|hpp|hxx)$")
```

Comment:
Do not export the variable and add a space between `|` and `grep`

Reply:
Without exporting variable, CI failed : https://github.com/powsybl/powsybl-iidm4cpp/actions/runs/244562520 :/
Maybe I missed something ?

Reply:
I don't understand why there is an error. On my Fedora, without the `export`, it works properly. We keep it like that